ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -13612,21 +13667,6 @@ def test_torch_tensor_xlogy(
     )
 
 
-# xlogy_
```

Comment:
please revert this, it was somehow misplaced above

Reply:
which files i should remove ?